.gitlab-ci.yml

```diff
@@ -245,7 +245,7 @@ run-certification-tests:
             echo "Deploying to $DEVICE_UNDER_TEST"
             tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
         fi
-      - /easymesh_cert/run_test_file.sh -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
+      - /easymesh_cert/run_test_file.sh -v -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
```

Comment:
I don't know if run_test_file.sh script has a "long options" for parameters, but if it does I prefer using the long options in scripts and use the short ones when typing in shell. In this case I would guess it is --verbose.
This way maintenance is easier and simpler as future readers do not need to guess or to go and look for meaning in the script.

Reply:
I tend to agree, but `-v` is quite standard and since we're using short options now I prefer to keep it.